sdk/spring/azure-spring-cloud-trace-sleuth/src/main/java/com/azure/spring/tracing/sleuth/implementation/HttpTraceUtil.java

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.tracing.sleuth.implementation;
+
+import org.springframework.cloud.sleuth.Span;
+
+public final class HttpTraceUtil {
+
+    private static final String STATUS_100 = "Continue";
```

Comment:
I don't think you need to define these stuff by youslef, how about using something existing, like `org.springframework.http.HttpStatus`

Reply:
OK, the dependency `org.springframework:spring-web` will be added.